src/core_plugins/console/api_server/es_6_0/reindex.js

```diff
@@ -1,18 +1,10 @@
 export default function (api) {
 
-  api.addEndpointDescription('_post_reindex', {
+  api.addEndpointDescription('reindex', {
     methods: [ 'POST' ],
     patterns: [
       '_reindex'
     ],
-    url_params: {
```

Comment:
The reindex api still [seems to still support these parameters](https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/reindex.json#L10-L37), why remove them?